yt_dlp/extractor/audioboom.py

```diff
@@ -27,14 +40,15 @@ class AudioBoomIE(InfoExtractor):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, video_id)
+        webpage = self._download_webpage(
+            'https://audioboom.com/posts/' + video_id, video_id)
 
         clip = None
 
         clip_store = self._parse_json(
-            self._html_search_regex(
-                r'data-new-clip-store=(["\'])(?P<json>{.+?})\1',
-                webpage, 'clip store', default='{}', group='json'),
+            unescapeHTML(self._html_search_regex(
+                r'data-react-class="V5DetailPagePlayer" *data-react-props=(["\'])(?P<json>{.+?})\1',
+                webpage, 'clip store', default='{}', group='json')),
             video_id, fatal=False)
```

Comment:
Use self._search_json

Reply:
Direct JSON parsing from the regex result does not work, because the JSON is coded with HTML entities.